app/models/casino/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class CASino::ApplicationRecord < ActiveRecord::Base
```

Comment:
What's the advantage of using this class over `ActiveRecord::Base` directly?

Reply:
@pencil Followed the Rails 5.0+ best practice. Also could serve as a better entry point for potential monkey-patching if needed.